src/components/calcite-dropdown-group/calcite-dropdown-group.scss

```diff
@@ -27,3 +27,9 @@
   cursor: default;
   @include font-size(-2);
 }
+
+:host .dropdown-separator {
```

Comment:
Is `:host` needed?

Reply:
Shouldn't need this for shadowed components.  I think this would only make sense for scoped css since it would concatenate the scoped host style with the text afterwards.

Reply:
No not needed just habit ✨ , removing.